vue.config.js

```diff
@@ -1,16 +1,68 @@
 const webpack = require('webpack')
 
+const iconVersion = 22 //fingerprint
+
 const qBittorrentPort = process.env['QBITTORRENT_PORT'] ?? 8080
 const vueTorrentPort = process.env['VUETORRENT_PORT'] ?? 8000
 const proxyTarget = process.env['QBITTORRENT_TARGET'] ?? 'http://localhost'
 
 module.exports = {
   pwa: {
+    manifestPath: 'manifest.' + iconVersion + '.json',
     name: 'VueTorrent', //PWApp name
     themeColor: '#597566', //PWA title bar color ( windows 10 PWA, android web browser and PWA address bar color )
     manifestOptions: {
+      start_url: 'https://m4rt.hakase.me:9980',
```

Comment:
is this url necessary?

Reply:
godd my test url, remove it. lol

Reply:
I made a mistake and forgot to delete it while branching again.